### PR TITLE
testing/dhcpdump - updated aport

### PR DESCRIPTION
**Pull request by archey**
updated apkbuild for dhcpdump to correctly build on musl

### CODE REVIEW

File: testing/dhcpdump/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        make
+}
+
+package() {
+        cd "$srcdir/$pkgname-$pkgver"
+        install -Dm755 dhcpdump "$pkgdir"/usr/bin/dhcpdump
+        install -Dm644 dhcpdump.8 "$pkgdir"/usr/share/man/man8/dhcpdump.8
+}
+
+md5sums="099c786997c424f196414f9575f1fb90  dhcpdump-1.8.tar.gz
```
Comment (Shizmob) [inline testing/dhcpdump/APKBUILD]: `md5sums=` and `sha256sums=` are unused and thus unneeded.

File: testing/dhcpdump/dhcpdump-fixes.patch [hunk 2]
```diff
@@ -0,0 +1,112 @@
+--- dhcpdump-1.8.orig/dhcpdump.c
```
Comment (Shizmob) [inline testing/dhcpdump/dhcpdump-fixes.patch]: Could you write a comment at the start of the file what the patch does and why it's needed?

File: testing/dhcpdump/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+# Contributor: Arch3y <arch3y@riseup.net>
```
Comment (Shizmob) [inline testing/dhcpdump/APKBUILD]: If you're going to moved it from `unmaintained`, I'm expecting you to take up maintainership as well. :)

**Comment by clandmeter** [comment]
Closed due to inactivity.

**Pull request closed without merging**